test/routes/Settings/routes/Account/components/test-account.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
```

Comment:
With how large this test file is, have you considered using a second layer of `describes` to break the tests even more? This would allow for a better understanding of what is going on and more organization.